miner/worker.go

```diff
@@ -468,6 +469,14 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 			txs.Pop()
 			continue
 		}
+		daBytesAfter := new(big.Int)
+		if ltx.DABytes != nil && miner.config.MaxDABlockSize != nil {
+			daBytesAfter.Add(blockDABytes, ltx.DABytes)
+			if daBytesAfter.Cmp(miner.config.MaxDABlockSize) > 0 {
+				txs.Pop()
```

Comment:
Let's add a log here that we're ignoring this transaction. In other cases, trace is used, which may be too low. Maybe debug level?

Reply:
We should also log when we drop txs that go over the individual limit.

Reply:
done